eras/shelley/impl/src/Cardano/Ledger/Shelley/API/Types.hs

```diff
@@ -50,7 +50,7 @@ import Cardano.Ledger.Keys as X
     Hash,
     KESignable,
     KeyHash (..),
-    KeyPair (..),
+    -- KeyPair (..),
```

Comment:
I think the right thing to do here is to make a new type, also called `KeyPair`, and deprecate it with a helpful message telling people where to find the new one.